src/test/java/tests/TurboMilestoneTest.java

```diff
@@ -34,6 +32,45 @@ public void turboMilestoneTest() {
     }
 
     @Test
+    public void getDueDateComparator_combinationOfDifferentMilestones_correctMilestoneOrdering() {
```

Comment:
Currently quite a few test methods use camelCase (e.g. [1](https://github.com/HubTurbo/HubTurbo/blob/master/src/test/java/tests/LogicTests.java#L95), [2](https://github.com/HubTurbo/HubTurbo/blob/master/src/test/java/tests/LogicTests.java#L115), [3](https://github.com/HubTurbo/HubTurbo/blob/master/src/test/java/tests/LogicTests.java#L95)). 

Should I still change it to Pascal case?